tests/scripts/all.sh

```diff
@@ -498,6 +500,11 @@ make
 
 msg "test: cmake 'out-of-source' build"
 make test
+
+msg "build: create and build mbed OS examples" # ~ 2h 17 min
+mkdir -p mbed-cli
+$MBEDTLS_ROOT_DIR/tests/scripts/mbed-build.sh mbed-cli
```

Comment:
@andresag01 OK, but I am not using pyuthon for this test, AFAIK
I will add mbed to the check tools in line 212